
# HG changeset patch
# User Alex Szpakowski <slime73@gmail.com>
# Date 1541897783 14400
# Node ID 5a95fbfd3617b3a309cd3f3a37143bde98872e28
# Parent  eb60e952b13fcd5fd00bafdaf1f209a5d3fb9d73
cocoa: fix building with the macOS 10.7 SDK (thanks Riccardo!)

Fixes bug #4368

diff -r eb60e952b13f -r 5a95fbfd3617 src/video/cocoa/SDL_cocoawindow.m
--- src/video/cocoa/SDL_cocoawindow.m	Sat Nov 10 16:15:48 2018 -0400
+++ src/video/cocoa/SDL_cocoawindow.m	Sat Nov 10 20:56:23 2018 -0400
@@ -1185,7 +1185,7 @@
     /* Force the graphics context to clear to black so we don't get a flash of
        white until the app is ready to draw. In practice on modern macOS, this
        only gets called for window creation and other extraordinary events. */
-    self.layer.backgroundColor = NSColor.blackColor.CGColor;
+    self.layer.backgroundColor = CGColorGetConstantColor(kCGColorBlack);
     ScheduleContextUpdates((SDL_WindowData *) _sdlWindow->driverdata);
     SDL_SendWindowEvent(_sdlWindow, SDL_WINDOWEVENT_EXPOSED, 0, 0);
 }

